env.example

```diff
@@ -54,3 +55,7 @@ BLOG_INACTIVE_TIME=360
 
 # GITHUB_TOKEN is used when fetching data from the GitHub API
 GITHUB_TOKEN=
+
+# REDIS_VERSION is the version of the Redis in the docker-compose file
+# Redis most stable version with an image in the Docker Hub; https://redis.io, https://hub.docker.com/_/redis
+REDIS_VERSION=5.0.7
```

Comment:
I don't think we need this in the env, it should probably go in the Dockerfile.

Reply:
I thought we might let the developer change the Redis without touching Dockerfiles, but you are right, it doesn't make sense to have it here. I will move this to its own file.

Reply:
If someone wants a custom Redis version, they can install it separate from Docker.  If they use Docker, we'll give them whatever version we specify in our image.

Reply:
@humphd should we have it `redis:latest` same as `node:latest` or I need to keep the version `5.0.7`
There are some comments around `latest` saying that it's not the best choice, but `latest` makes it easier to maintain. So what would you suggest for both?

Reply:
For node, we want `lts` (long term stable vs. latest).  It's the "stable" version vs. the latest.

For Redis, let's use `latest`, which is really the [current stable version](https://github.com/docker-library/redis/blob/d42494ab2d96070c8d83f37a7542fbbffd999988/5.0/Dockerfile#L42).  Then we can get rid of this, and let Docker/Redis manage this for us.

Reply:
I didn't know that acronym. Thanks for the clarification.